server/modules/email/emailModule.js

```diff
@@ -95,3 +98,22 @@ export const itEmailTransport = (developerName, developerEmail, developerLocatio
 
   return sendMail(mailOptions);
 };
+
+export const opsOffBoardingEmailTransport = (
+  developerName,
+  partnerName,
+  developerEmail,
+  developerLocation,
+  partnerLocation,
+  rollOffDate,
+) => {
```

Comment:
I think this can be refactored.
*Problem*: to many arguments
*Solution*: You can use spread operator and destructuring as follows:

```
export const opsOffBoardingEmailTransport = (...args) => {
    const {developerName, partnerName, developerEmail, developerLocation, partnerLocation, rollOffDate} = args;
     // your code
};

Reply:
Ok, but wouldn't code climate still complain about similar code blocks?

Reply:
I don't think so

Reply:
If we would be going with @nzediegwu1 suggestion of destructuring the arguments passed to the function, please ensure to use JSDoc commenting to indicate the parameters needed by the function.

Reply:
@machariamarigi How can we test this locally?

Reply:
@daumie Looking for email testing solutions